anomalydetection/features.py

```diff
@@ -20,3 +20,12 @@ def lag_time_series(time_series: pd.Series, lags):
         lagged_time_series[str(lag)] = time_series.shift(lag)
 
     return pd.concat(lagged_time_series, axis=1)
+
+
+def create_dataset(X, y, time_steps=1):
```

Comment:
I'm thinking a test for this would be good, even though it is of course quite straight forward.

Reply:
Sure. Just added one